pyproject.toml

```diff
@@ -75,7 +75,7 @@ include = [
     "./apps/prairielearn/elements/pl-hide-in-panel",
     "./apps/prairielearn/elements/pl-integer-input",
     "./apps/prairielearn/elements/pl-manual-grading-only",
-    "./apps/prairielearn/elements/pl-matrix-latex",
```

Comment:
We still need to typecheck `matrix-latex` as well.

Reply:
Readded. I think I might have broken the laws of physics when trying to merge latest changes into this branch after 3 weeks of inactivity.